Add fading on/off, less integer divisions.

benallard opened the pull request:
Fix issues with integer divisions.

Add fading on/off

benallard committed:
Add fading on/off, less integer divisions.

polhenarejos commented:
It was corrected in [a816b6f](https://github.com/polhenarejos/pico-keys-sdk/commit/a816b6f747604c3430faadb66aefba067326f8ed)

Added options to enable/disable dimming (fading) and max brightness through pico-fido-tool/pico-hsm-tool PHY option. By default is disabled and max brightness for compatibility with single led boards. Those with ws2812 can be enabled through cli tools.

benallard commented:
Okay, sure. Fee remarks from my side:- if you set the same brightness in all modes, there is not much need of setting it explicitly per mode.- I had added the possibility to switch fading on/off per mode. Might be interesting if you want to attract the attention with “real” blinking.Thank you for your work on this!On 28. Sep 2024, at 11:16, Pol Henarejos ***@***.***> wrote:﻿
It was corrected in a816b6f
Added options to enable/disable dimming (fading) and max brightness through pico-fido-tool/pico-hsm-tool PHY option. By default is disabled and max brightness for compatibility with single led boards. Those with ws2812 can be enabled through cli tools.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
